src/stories/Search.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { Search } from '../components/Search.tsx'
+
+const meta: Meta<typeof Search> = {
+    title: 'Search',
+    component: Search,
+    parameters: {
+        layout: 'centered',
+    },
+    tags: ['autodocs'],
+    argTypes: {
+        initialQuery: { control: 'text', description: 'Initial Query' },
+        onSearch: { action: 'searched' },
+    },
+}
+
+export default meta
+
+type Story = StoryObj<typeof Search>
+
+export const Default: Story = {
+    args: {
+        initialQuery: 'Search Query',
+        onSearch: (query: string) => {
+            console.log('Searching for:', query)
+        },
```

Comment:
There is no need to provide a logging callback - this functionality is embedded in Storybook and is called "Actions". You already described an action in `argTypes`, so if you remove your manual callback you will see that every time after clicking the Search button on the story page the action is logged in the Actions tab.  
![image](https://github.com/eduard-bagdagulyan/react-global-mentoring-2024/assets/157795858/3fa682fd-3422-4016-9c39-fbac9206e362)